openelevationservice/server/api/querybuilder.py

```diff
@@ -8,10 +8,15 @@
 from openelevationservice.server.api.api_exceptions import InvalidUsage
 
 from geoalchemy2.functions import ST_Value, ST_Intersects, ST_X, ST_Y # ST_DumpPoints, ST_Dump, 
-from sqlalchemy import func, literal_column, case
+from sqlalchemy import func, literal_column, case, text
 from sqlalchemy.types import JSON
 from sqlalchemy.dialects.postgresql import array
 
+
+# Importing all the code that parallelizes elevation requests
```

Comment:
It is not necessary to comment that libraries will be imported.

Reply:
Done. Deleted.